packages/next/lib/chalk.ts

```diff
@@ -1,6 +1,6 @@
 let chalk: typeof import('next/dist/compiled/chalk')
 
-if (typeof window === 'undefined') {
+if (!process.browser) {
```

Comment:
Why we change back to `process.browser` here 🤔

Reply:
Because the edge runtime is “web”, and it doesn’t have `window` too. So the current condition here is wrong…